TUM-Dev/NavigaTUM
# chore:sqlx migration calendar

## Proposed Changes (include Screenshots if possible)

- finished the diesel -> sqlx migration

## How to test this PR

1. Run testcases
2. see if it downloads without problems

## How has this been tested?

-
-

## Checklist
